changelog/september2024/2024-09-20-jobs-fixed-created-at-and-started-at.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Created At and Started At
+status: fixed
+author:
+    fullname: 'Join the #serverless-jobs channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-09-20
+category: serverless
+product: jobs
+---
+
+On Scaleway Console on Serverless Jobs Runs on some places the created_at date was displayed instead of started_at date.
```

Comment:
```suggestion
In the Scaleway console the `created_at` date appeared instead of the `started_at` date for some Serverless Jobs runs. The bug is now fixed.
```